lib/config/include/hse/config/params.h

```diff
@@ -88,15 +57,15 @@ struct param_spec {
     size_t          ps_offset;
     size_t          ps_size;
     /* Converts a JSON node into the expected data */
-    param_converter_t ps_convert;
+    param_converter_t *ps_convert;
     /* Validates data just after ps_convert() */
-    param_validator_t ps_validate;
+    param_validator_t *ps_validate;
     /* Validates relations after conversion, validation, and updating of all data */
-    param_relation_validator_t ps_validate_relations;
+    param_relation_validator_t *ps_validate_relations;
     /* Returns a JSON string representation of the value */
-    param_stringify_t ps_stringify;
+    param_stringify_t *ps_stringify;
     /* Returns a JSON representation of the parameter */
-    param_jsonify_t ps_jsonify;
+    param_jsonify_t *ps_jsonify;
```

Comment:
Hard to read, better if short comment followed the decl on the same line...

Reply:
I see your point